src/nunit.uikit/Controls/TipWindow.cs

```diff
@@ -177,6 +177,13 @@ protected override void OnLoad(System.EventArgs e)
 
 			Size sizeNeeded = Size.Ceiling( g.MeasureString( tipText, Font, layoutArea ) );
 
+            // If the needed width is smaller than that of the original label,
```

Comment:
ugh, i should have chose the bottom of the change instead of the top.. then this woudl probably make more sense. I wish that GitHub allowed for multi-line comments